HOSTING.md

```diff
@@ -7,17 +7,21 @@ We keep adding new hosting providers regularly - please let us know if you need
 
 To run a hosted version of `vectara-answer` you would need to have two files: `.env` and `queries.json`.
 
-To prepare the `.env` file, which combines the information from `config.yaml` and `secrets.toml`, run the following command:
+To prepare the `.env` file, which combines the information from `config.yaml` and `secrets.toml`, perform the following:
+1. `cd` to the main `vectara-answer` folder, where your `secrets.toml` file resides
+2. Execute `python3 docker/prepare_config.py /path/to/<YOUR-CONFIG-FOLDER> <PROFILE>`.
 
-`python3 prepare_config.py /path/to/<YOUR-CONFIG.yaml> <toml profile (usually default)>`
+The `prepare_config.py` script acceptes two arguments
+- The first is the path to your configuration folder (where `config.yaml` and `queries.json` reside)
+- The `<PROFILE>` refers to the profile name you used in `secrets.toml`. If you didn't create a new profile, it likely is called `default`.
 
 At the end of this process you should have two files ready to go:
-1. `.env` generated by the above python3 command
-2. `queries.json` that specifies the queries for your application.
+1. `.env` in the main `vectara-answer` folder, generated by the above python3 command
+2. `queries.json` (in your original configuration folder) that specifies the queries for your application.
```

Comment:
I suggest removing the parens here, for consistency with the lack of parens in the first list item.